### PR TITLE
Add Lithuanian Contributing file

### PR DESCRIPTION
**Pull request by GintasS**

### CODE REVIEW

**Comment by wopian** [comment]
The `Ką man reikia patikrinti?` section was recently updated and needs re-translating to match the current translation. 

You can find the changes made to the English CONTRIBUTING at https://github.com/chrislgarry/Apollo-11/commit/16aa2154b23dda648a2ab568734e11627af8448b

**Comment by GintasS** [comment]
> The `Ką man reikia patikrinti?` section was recently updated and needs re-translating to match the current translation.
> 
> You can find the changes made to the English CONTRIBUTING at [16aa215](https://github.com/chrislgarry/Apollo-11/commit/16aa2154b23dda648a2ab568734e11627af8448b)

Will change.

**Comment by GintasS** [comment]
Updated to match the changes.

**Comment by wopian** [comment]
Thank you :)

File: CONTRIBUTING.lt.md [hunk 1]
```diff
@@ -0,0 +1,143 @@
+# Prisidėti
+
+🎌
+[Čeština][CZ],
+[Deutsch][DE],
+**Lietuvių**,
```
Comment (wopian) [inline CONTRIBUTING.lt.md:6]: Alphabetical order (After Kurdi)

**Review by wopian — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.lt.md (hunk 1 above)
Comment (GintasS) [reply CONTRIBUTING.lt.md]: Fixed.

**Review by wopian — APPROVED** [review]

**Pull request merged**